Add config and organize the project

itsikm opened the pull request.

Commit:
- Add Jira issue widget + job
- Add Jira status widget + job

Commit:
- Add Github pull requests job

Commit:
- Add config.js & organize the project

guy-mograbi-at-gigaspaces commented on .gitignore:
```diff
@@ -1,3 +1,4 @@
-node_modules
\ No newline at end of file
+node_modules
+meConf.json
```
no need to ignore meConf.json. 
meconf.json should be under "dev" folder.

guy-mograbi-at-gigaspaces commented on config.js:
```diff
@@ -0,0 +1,26 @@
+}
+
+// Default Configuration
+var config = {
```
please put default config under "prod.json" - we can use it as reference for me.json.

guy-mograbi-at-gigaspaces commented on config.js:
```diff
@@ -0,0 +1,26 @@
+var meConf = null;
+
+try {
+    meConf = require('./meConf');
```
we talked about allowing to override me.json location with environment variable. need to add it.

guy-mograbi-at-gigaspaces commented on config.js:
```diff
@@ -0,0 +1,26 @@
+    meConf = require('./meConf');
+}
+catch(e) {
+    console.warn('(optional) meConf.json is missing.');
```
console? 
please use logger.

guy-mograbi-at-gigaspaces commented on jobs/jira-issues.job.js:
```diff
@@ -0,0 +1,68 @@
+        addUserIssueStatus(userKey, issue.fields.status.name.toLowerCase());
+    }
+    catch (e) {
+        console.log('addUserIssue:error', e, issue);
```
please use logger

Commit:
Organizing the project:
- Add prod.json as default config
- Add Github token to configuration
- Using logger instead of console
